returnn/tf/layers/basic.py

```diff
@@ -5260,7 +5260,9 @@ def get_out_data_from_opts(
       dim_tags += out_spatial_dims
     else:
       for i in range(len(filter_size)):
-        old_tag = old_spatial_dim_tags[i] if i < len(old_spatial_dim_tags) else None
+        old_tag = old_spatial_dim_tags[i].copy(
+                    same_as_self=False, description=old_spatial_dim_tags[i].description
```

Comment:
Formatting (indentation) is off.

But also, this does not make sense. You explicitly want that you have `same_as_self=True` here. Also, there should be no copy of the tag.